src/deliver-mobile/app-shield/intro.md

```diff
@@ -178,6 +178,10 @@ These are the values available in the **AppShield** configuration JSON.
 | global                          | JSON value   | iOS, Android | Settings in this section apply to both Android and iOS builds.                                    |
 | ios                             | JSON value   | iOS          | The key denoting values that apply to the iOS devices.                                            |
 
+## Verifying that you've successfully built an app with AppShield
+When you trigger a build with the Appshield enabled and correctly configured, **MABS** will include a **shield phase** in the building process of your mobile package. You can verify that your build passed through the shield phase by looking into your app’s build logs. The shield phase is logged after the build phase.
```

Comment:
Hi folks, I just noticed I forgot to add a link and a small bit of info to this piece of text. So, if possible consider this text instead: 

"When you trigger a build with the Appshield enabled and correctly configured, MABS will include a shield phase in the process of generating your mobile package. You can verify that your build passed through the shield phase by looking into your app’s build logs (see here [how to download the build log file of your mobile package](http://xn--you%20can%20verify%20that%20your%20build%20passed%20through%20the%20shield%20phase%20by%20looking%20into%20your%20apps%20build%20logs-x689f/)). 

The shield phase is logged after the build phase and it's the last phase of the build log file."

Reply:
I noticed I can Edit the pull request, so I submitted an Edit to this pull with these new changes.